package.js

```diff
@@ -1,33 +1,32 @@
 Package.describe({
   name: 'mdg:validated-method',
   summary: 'A simple wrapper for Meteor.methods',
-  version: '1.1.0',
+  version: '1.2.0',
```

Comment:
I wonder if we should bump this major version?

Reply:
I mainly ask because package pinning has changed a bit over the Meteor 1.x versions, but I think that was only affecting core packages, not third-party packages, and this package was never "core".

Reply:
I think we should be okay with the minor version bump, but if not a major bump sounds good.